tests/py/unit/fixtures.py

```diff
@@ -33,18 +39,110 @@
     "substrate_transformation": [[1.0, 0.0], [0.0, 1.0]],
     "strain": Strain([[0.004746364, -0.0, -0.0], [-0.0, 0.004746364, 0.0], [-0.0, 0.0, -0.0]]),
     "von_mises_strain": 0.001,
-    "termination": INTERFACE_TERMINATION,
+    "termination": INTERFACE_TERMINATION_PAIR,
 }
 INTERFACE_PROPERTIES_JSON = {
     "film_transformation": [[2.0, 0.0], [0.0, 2.0]],
     "substrate_transformation": [[1.0, 0.0], [0.0, 1.0]],
     "strain": [[0.004746364, -0.0, -0.0], [-0.0, 0.004746364, 0.0], [-0.0, 0.0, -0.0]],
     "von_mises_strain": 0.001,
     "termination": INTERFACE_TERMINATION_AS_STR,
-    "mean_abs_strain": 0.00105,
+    # "mean_abs_strain": 0.00105,
 }
 
 
 # Add properties to interface structure
 INTERFACE_STRUCTURE.interface_properties = INTERFACE_PROPERTIES_MOCK
-INTERFACE_STRUCTURE = interface_patch_with_mean_abs_strain(INTERFACE_STRUCTURE)
+
+# TODO: Use fixtures package when available
+SI_SUPERCELL_2X2X1 = {
+    "name": "Si8",
+    "basis": {
+        "elements": [
+            {"id": 0, "value": "Si"},
+            {"id": 1, "value": "Si"},
+            {"id": 2, "value": "Si"},
+            {"id": 3, "value": "Si"},
+            {"id": 4, "value": "Si"},
+            {"id": 5, "value": "Si"},
+            {"id": 6, "value": "Si"},
+            {"id": 7, "value": "Si"},
+        ],
+        "coordinates": [
+            {"id": 0, "value": [0.0, 0.0, 0.0]},
+            {"id": 1, "value": [0.125, 0.125, 0.25]},
+            {"id": 2, "value": [0.0, 0.5, 0.0]},
+            {"id": 3, "value": [0.125, 0.625, 0.25]},
+            {"id": 4, "value": [0.5, 0.0, 0.0]},
+            {"id": 5, "value": [0.625, 0.125, 0.25]},
+            {"id": 6, "value": [0.5, 0.5, 0.0]},
+            {"id": 7, "value": [0.625, 0.625, 0.25]},
+        ],
+        "units": "crystal",
+        "cell": [[6.697840473, 0.0, 3.867], [2.232613491, 6.314784557, 3.867], [0.0, 0.0, 3.867]],
+        "constraints": [],
+    },
+    "lattice": {
+        "a": 7.734,
+        "b": 7.734,
+        "c": 3.867,
+        "alpha": 60.0,
+        "beta": 60.0,
+        "gamma": 60.0,
+        "units": {"length": "angstrom", "angle": "degree"},
+        "type": "TRI",
+        "vectors": {
+            "a": [6.697840473, 0.0, 3.867],
+            "b": [2.232613491, 6.314784557, 3.867],
+            "c": [0.0, 0.0, 3.867],
+            "alat": 1,
+            "units": "angstrom",
+        },
+    },
+    "isNonPeriodic": False,
+    "_id": "",
+    "metadata": {"boundaryConditions": {"type": "pbc", "offset": 0}},
+    "isUpdated": True,
+}
+
+SI_SLAB = {
```

Comment:
Maybe explain which slab parameters this corresponds to either in a comment or in the name itself